jme3-core/src/plugins/java/com/jme3/export/binary/BinaryLoader.java

```diff
@@ -0,0 +1,78 @@
+package com.jme3.export.binary;
+
+import com.jme3.asset.AssetInfo;
+import com.jme3.asset.AssetManager;
+import com.jme3.export.InputCapsule;
+import com.jme3.export.JmeImporter;
+import com.jme3.export.Savable;
+
+import java.io.IOException;
+import java.util.ArrayDeque;
+import java.util.Deque;
+
+/**
+ * The default loader to load binaries files.
+ *
+ * @author JavaSaBr
+ */
+public class BinaryLoader implements JmeImporter {
```

Comment:
Why does this need to extend `JmeImporter? Shouldn't it implement `AssetLoader` instead?